cmd/csi-snapshotter/main.go

```diff
@@ -141,7 +142,7 @@ func main() {
 	glog.V(2).Infof("CSI driver name: %q", *snapshotter)
 
 	// Check it's ready
-	if err = waitForDriverReady(csiConn, *connectionTimeout); err != nil {
+	if err = waitForDriverReady(csiConn, *operationTimeout); err != nil {
```

Comment:
Passing in operation timeout here seems awkward. Should waitForDriverReady just poll forever instead?

Reply:
This is related: https://github.com/kubernetes-csi/csi-lib-utils/pull/17
cc @jsafrane